### ISSUE TITLE
Accessibility Fail

### ISSUE THREAD

**Issue by rgraciadp** [stephenscaff/read-smore#25]
This plugin needs accessibility in order for it to be widely used in production sites. Right now there is zero keyboard support for tabbing to the link and expand the text. Please add that so we can use it again. Otherwise works great. Thank you!

**Comment by stephenscaff** [comment]
Yeah, fair enough. Will take a look.

**Comment by stephenscaff** [comment]
Okay, added a11y support for : 

- tabbing to link element
- Ability to toggle with enter key on link element
- aria-expanded tag on link element. 

Open to any further suggestions. Gonna close this, but feel free to let me know if I should do anything else.

Thanks.
